corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -51,11 +51,16 @@
         $("#archive-form").submit(function() {
             document.getElementById('archive-form-btn').disabled=true;
             document.getElementById('archive-spinner').style.display='inline-block';
-            ga_track_event('Reports', 'Case History', 'Archive this form', {
-                'hitCallback': function () {
+
+            // _.after(2,...) means the callback will only be called after *both* analytics
+            // functions have finished.
+            var analyticsCallback = _.after(2, function() {
```

Comment:
If GA/Kissmetrics is blocked, i.e. if they are not loaded at all, they don't call callbacks. I fixed this for GA [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/includes/analytics_google.html#L113). Looks like kissmetrics has same problem. Commented on https://github.com/dimagi/commcare-hq/pull/7087